### PR TITLE
MDCT-94: Configure Cypress testing

### PR DESCRIPTION
**Pull request by braxex**
# Description

The [ticket](https://qmacbis.atlassian.net/browse/MDCT-94) was just to set up Cypress testing in the repo, which it turns out was already done thanks to QMR. There was a bunch of stuff that needed to be ripped out and cleaned up, though.

## How to test

To run integration tests: 
  - in project root dir
  - `yarn && yarn test`
  - select `commonPages.spec.js` in the Cypress window that appears
  - watch the testing occur

Note: There are some minor accessibility issues that will be fixed later when we actually have designs built for MCR. You'll also see some API call failures, which are not a big deal at this point because that will also be changed later. All that's really being tested is the login and homepage render. 👍

## Dependencies

No updated dependencies.

# Get it done

1. You will review it (thanks!)
2. I will merge it (good job, me).

## Code authors checklist

- [x] I have performed a self-review of my code
- [x] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [x] ~~Necessary analytics were added, or no new analytics were required~~
- [x] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [x] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

File: tests/cypress/support/pages/Homepage.js [hunk 1]
```diff
@@ -1,76 +1,56 @@
...
-  }
-
-  clickAdultCoreSetMeasures() {
-    cy.wait(3000);
-    cy.scrollTo("top");
-    cy.xpath(adultCoreSetMeasures).click({ force: true });
-    //cy.xpath(adultCoreSetMeasures).click();
+  verifyLogos() {
+    cy.xpath(appLogo).should("be.visible");
+    cy.xpath(footerMDCTLogo).should("be.visible");
```
Comment (juliuscaesar) [inline tests/cypress/support/pages/Homepage.js:19]: This should be `footerMdctLogo` (or capitalized on line 8)

**Comment by juliuscaesar** [comment]
Are you adding the tests in a nested folder for a reason? `tests/cypress/cypress/integration/`? I have them in `tests/cypress/tests` but can nest them in a second `cypress` directory or rename the `tests` to `integration`. For some reason I thought I saw you modify `integration` to `tests` so that's what I did.

**Comment by braxex** [comment]
> Are you adding the tests in a nested folder for a reason? `tests/cypress/cypress/integration/`? I have them in `tests/cypress/tests` but can nest them in a second `cypress` directory or rename the `tests` to `integration`. For some reason I thought I saw you modify `integration` to `tests` so that's what I did.

@juliuscaesar I did the opposite, actually. There was originally a `tests/cypress/cypress/integration` dir, but I got rid of it in favor of `tests/cypress/tests` 'cause that makes a lot more sense. So you're on the right track with yours. 👍

File: tests/cypress/support/pages/Homepage.js (hunk 1 above)
Comment (braxex) [reply tests/cypress/support/pages/Homepage.js:19]: Good catch, thanks.

**Review by juliuscaesar — APPROVED** [review]

**Pull request merged**